grails-app/assets/javascripts/streama/controllers/admin-reports-ctrl.js

```diff
@@ -18,13 +21,15 @@ angular.module('streama').controller('adminReportsCtrl', [
 
     function pageChanged () {
 
-      var newOffset = vm.maxPerPage*(vm.pagination.currentPage-1);
-      vm.loadReports({max: vm.maxPerPage, filter: vm.listFilter, offset: newOffset});
+      currentOffset = vm.maxPerPage*(vm.pagination.currentPage-1);
+      vm.loadReports({max: vm.maxPerPage, filter: vm.listFilter, offset: currentOffset, sort: vm.sortAndOrderBy.sort, order: vm.sortAndOrderBy.order});
```

Comment:
if you are on page 2, for example, and then you use one of the filter-buttons, then nothing gets shown, because internally it still sends the old offset of 15. This needs to be resettet to 0 when filtering.

Reply:
same for sorting

Reply:
I think it actually behaves correctly for sorting atm. I'm looking into the filter buttons but I think it's not as simple as resetting the offset to 0

Reply:
Changed the offset to 0 now and always reset page to 1 when changing filters. Sorting should be behaving correctly though I believe. If not, let me know.